### DISCUSSION TITLE
Create a Parent Prefix for all VRFs

### DISCUSSION THREAD

**Discussion by mickm2023** [netbox-community/netbox#14068] [Q&A]
Hi All,

Looking for some help with creating a parent Prefix for all VRFs so we can see available child prefixes no matter what VRF it belongs to.

Example.
10.0.0.0/8 (vrf : Global)
- 10.0.1.0/24 (vrf : Green)
- 10.0.2.0/24 (vrf : Blue)
- 10.0.3.0/24 (vrf : Global)

How it is now. (This will show 10.0.1.0/24 & 10.0.2.0/24 as available)
10.0.0.0/8 (vrf : Global)
- 10.0.3.0/24 (vrf : Global)

10.0.1.0/24 (vrf : Green)
10.0.2.0/24 (vrf : Blue)

I know this would kinda destroy the purpose of VRFs in general. But with the current design of our datacenter we don't have duplicated Prefixes across our infrastructure. Which is why we want to be able to see all available prefixes under 1 Parent no matter if it is in green, blue or red VRF.

**Comment by PieterL75** [comment]
See https://github.com/netbox-community/netbox/discussions/6533
